src/utils/samlUtils.ts

```diff
@@ -24,6 +24,7 @@ import * as xmlCrypto from "xml-crypto";
 import { Builder, parseStringPromise } from "xml2js";
 import { DOMParser } from "xmldom";
 import { SPID_LEVELS, SPID_URLS, SPID_USER_ATTRIBUTES } from "../config";
+import { EventTracker } from "..";
```

Comment:
Is he taking the EventTracker from the directory itself? How is this working?

Reply:
when no name is provided it assumes you're referencing the file named index.
in this case it takes the type `EventTracker` from the `index.ts` file in the parent directory

Reply:
ok it's fine.